imports/plugins/core/files/server/methods.js

```diff
@@ -46,6 +50,38 @@ export async function insertMedia(fileRecord) {
 export async function removeMedia(fileRecordId) {
   check(fileRecordId, String);
 
+  const mediaRecord = await MediaRecords.findOne({ _id: fileRecordId });
```

Comment:
I'm going to do a second PR to convert all of these to GraphQL mutations. For now since it was working I figured I'd avoid messing with it more than necessary. This is a weird file since it runs in Meteor context and has a mixture of things being awaited as well as things that use Meteor Fibers to be synchronous, but Meteor seems to figure it all out somehow.